attorch/regularizers.py

```diff
@@ -3,10 +3,13 @@
 import numpy as np
 import torch
 from itertools import product
+from torch.nn import functional as F
 
+# def laplace():
+#     return np.array([[0.25, 0.5, 0.25], [0.5, -3.0, 0.5], [0.25, 0.5, 0.25]]).astype(np.float32)[None, None, ...]
 
 def laplace():
-    return np.array([[0.25, 0.5, 0.25], [0.5, -3.0, 0.5], [0.25, 0.5, 0.25]]).astype(np.float32)[None, None, ...]
+    return np.array([[0,-1, 0], [-1, 4, -1], [0, -1, 0]]).astype(np.float32)[None, None, ...]
```

Comment:
I'll be concerned that this change will make existing trained networks invalid. What's the basis for changing the smoothening filter?

Reply:
It won't make them invalid. They just won't train to the same state with the same parameters. I changed it to make it consistent with the filter that Alex uses.